Source/Rows/DatePickerRow.swift

```diff
@@ -53,6 +53,18 @@ open class DatePickerCell: Cell<Date>, CellType {
         height = { UITableView.automaticDimension }
         datePicker.datePickerMode = datePickerMode()
         datePicker.addTarget(self, action: #selector(DatePickerCell.datePickerValueChanged(_:)), for: .valueChanged)
+
+        #if swift(>=5.2)
+            if #available(iOS 14.0, *) {
+                #if swift(>=5.3) && !(os(OSX) || (os(iOS) && targetEnvironment(macCatalyst)))
+                    datePicker.preferredDatePickerStyle = .inline
```

Comment:
@mats-claassen Why using `.inline` here? Should this not be exclusive for `DateInlineRow`?

I explicitly try to avoid the new iOS 14 DatePicker with inline style because the time selection is covered by the keyboard when being edited. Maybe a contentInset issue.